### PR TITLE
Add TxHash Decoder

### PR DESCRIPTION
**Pull request by edwin082**
This page can help you to utilize and test the "caver.transaction.getTransactionByHash()" function.

### CODE REVIEW

File: src/views/txHashDecoder.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import React, { Component } from "react";
+import { Button, Card, CardHeader, CardBody, Row, Col, Label } from "reactstrap";
+import '../../assets/css/black-dashboard-react.css';
+import Caver from 'caver-js'
+import { networkLinks } from '../constants/klaytnNetwork'
+import Column from "../components/Column"
+let caver;
+
+class txHashDecoder extends Component {
```
Comment (Yeonju-Kim) [inline src/views/txHashDecoder.js]: Could you change the component name to "TxHashDecoder"?

File: src/views/txHashDecoder.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                                </select>
+                            </Col>
+                        </Row>
+                        </CardHeader>
+                        <CardBody>
+                            <Row>
+                                <Col md = "12">
+                                    <Label>TxHash</Label>
+                                    <p style={{color:"#6c757d"}}>
+                                    Enter the TxHash.
```
Comment (Yeonju-Kim) [inline src/views/txHashDecoder.js]: Please change TxHash to "Transaction Hash".

File: src/views/txHashDecoder.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    render() {
+        const { txHash, result, rawResult } = this.state;
+        return (
+            <div>
+                <Column>
+                    <Card>
+                        <CardHeader>
+                            <h3 className="title">TxHash Decoder</h3>
+                            <p style={{color:"#6c757d"}}>
+                            The tool was designed to get a transaction from the transaction hash(TxHash).
```
Comment (Yeonju-Kim) [inline src/views/txHashDecoder.js]: How about changing this sentence to "On this page, you can query a transaction by transaction hash from the Klaytn network to get a caver transaction instance." ? Please let me know your thought.

**Review by Yeonju-Kim — CHANGES_REQUESTED** [review]

File: src/views/txHashDecoder.js (hunk 3 above)
Comment (edwin082) [reply src/views/txHashDecoder.js]: I think, that sentence is more better to understand

**Pull request closed without merging**